http/http.go

```diff
@@ -0,0 +1,88 @@
+package http
```

Comment:
our package has the same name as net/http :(

Reply:
^ I brought this up, thinking it would cause issues but was happy to see how things go when we start building more of the app.

Reply:
I did some testing around this by calling our library from `main` and it was working just fine.

If I was doing
``golang
import "net/http"

...

http.Get(...)

```
was calling golang's http client and if I was doing:
``golang
import "github.com/bluetel/go-sf/http"

...

http.Get(...)

```

This was calling our http library. I can't see a massive issue since you can always aliase them right?

Reply:
Yeah I mean its still going to work, its just a little ambiguous is all - like in our http package, its hard to see which http you're using